libkbfs/crypto_common.go

```diff
@@ -69,11 +69,11 @@ func (c CryptoCommon) MakeRandomBranchID() (BranchID, error) {
 }
 
 // MakeMdID implements the Crypto interface for CryptoCommon.
-func (c CryptoCommon) MakeMdID(md *BareRootMetadata) (MdID, error) {
+func (c CryptoCommon) MakeMdID(md ImmutableBareRootMetadata) (MdID, error) {
```

Comment:
This function implicitly assumes that the given object is serializable. Maybe IBRMD this method needs to belong to `IBRMD`.